app/assets/stylesheets/_preferences.scss

```diff
@@ -0,0 +1,82 @@
+.preferences-view {
```

Comment:
Styling that will likely only be used on one component shouldn't be extracted on a separate class, but we should rather keep the utility classes in the markup itself. For example, if the styling for .preferences-view is only used in that component, then we wouldn't have a separate preferences-view class, we'd just use all the utility classes in the JSX markup. If there's styling that's likely going to be repeated in several components, then we can extract them to its own class. Here's a bit more info about this approach https://tailwindcss.com/docs/extracting-components

Reply:
Yep, fixed.